force-app/main/default/flexipages/GetStarted.flexipage-meta.xml

```diff
@@ -24,7 +24,7 @@
         <name>region1</name>
         <type>Region</type>
     </flexiPageRegions>
-    <flexiPageRegions>
```

Comment:
This component will never be rendered. Comment it out so the extra spacing can be removed